opennms-base-assembly/src/main/filtered/etc/snmp-interface-poller-configuration.xml

```diff
@@ -1,4 +1,4 @@
-<snmp-interface-poller-configuration xmlns="http://xmlns.opennms.org/xsd/config/snmpinterfacepoller" threads="30" service="SNMP">
+<snmp-interface-poller-configuration xmlns="http://xmlns.opennms.org/xsd/config/snmpinterfacepoller" threads="30" service="SNMP" suppressInitializationEvent="false">
```

Comment:
We can omit this change, since the attribute is optional.

Reply:
Config option removed in 8350b4e